src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -99,7 +129,27 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 		if (!terminal || !commandDetection) {
 			return;
 		}
-		this._register(commandDetection.onCommandFinished(command => this._resolveQuickFixes(command)));
+		this._register(commandDetection.onCommandFinished(command => {
+			if (this._fixesAvailable) {
+				if (this._fixesShown) {
+					if (this._expectedCommands?.includes(command.command)) {
+						this._logService.debug(QuickFixCommandResultTelemetryEvent.QuickFixExecutedManually);
+						this._telemetryService?.publicLog2<{ classification: 'SystemMetaData'; purpose: 'FeatureInsight' }>(QuickFixCommandResultTelemetryEvent.QuickFixExecutedManually);
+					} else {
```

Comment:
Won't this also fire when a quick fix is executed via the dropdown? A quick fix should fire `onCommandFinished` as well

Reply:
not sure what you mean - would suggest testing it w debug level logging on bc does what i'm expecting